x-pack/auditbeat/module/system/package/package.go

```diff
@@ -215,7 +215,9 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		}
 	case darwin:
 		if _, err := os.Stat(homebrewCellarPath); err != nil {
-			return nil, errors.Wrapf(err, "error looking up %s - is Homebrew installed?", homebrewCellarPath)
+			ms.log.Errorf("Failed to start package dataset on macOS. Error looking up %s - is Homebrew installed? Error: %v",
```

Comment:
What if Homebrew is installed while Auditbeat is running? The user needs to restart Auditbeat to get monitoring.

Another approach would be to run the MetricSet and have is start reporting information if when the Homebrew database comes into existence.

Reply:
Makes sense, I've changed to that.